app/processing/payment/get-period-from-schedule.js

```diff
@@ -0,0 +1,10 @@
+const getPeriodFromSchedule = (totalValue, totalPayments, paymentValue, schedule, increment, unit) => {
```

Comment:
Appreciate tht your higher level integration tests cover this return, but do you think we should unit test this also, given yer comment below on how this will break with PPA -- good to have quick and less convolved way of how its broke it 😆

Reply:
Yes sensible idea